llvm/test/Transforms/Inline/inline-switch-default-2.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
-; RUN: opt %s -S -passes=inline -inline-threshold=21 | FileCheck %s
+; RUN: opt %s -S -passes=inline -inline-threshold=11 | FileCheck %s
```

Comment:
Please fix the FileCheck lines in this file; there's a bunch of checks for stuff like "LOOKUPTABLE" which don't correspond to a RUN line.  (Maybe push this separately.)

Reply:
Updated: https://github.com/llvm/llvm-project/commit/971ec1f0eea324d4a1eec6709e2c97e1798a6002